fabric-events-interaction-v0/src/main/java/net/fabricmc/fabric/mixin/event/interaction/MixinClientPlayerInteractionManager.java

```diff
@@ -16,6 +16,11 @@
 
 package net.fabricmc.fabric.mixin.event.interaction;
 
+import net.fabricmc.fabric.api.event.player.*;
```

Comment:
You know you can set IJ to not static import and order imports properly.

Reply:
haha, ill be switching the vscode soon completely, it shouldn't happen there